libcxx/test/benchmarks/containers/associative/hash_table_find.bench.cpp

```diff
@@ -0,0 +1,70 @@
+#include <unordered_set>
+#include <string>
+#include <random>
+#include <vector>
+
```

Comment:
Looks like `// UNSUPPORTED: c++03` should be added here (see also other test files).

Reply:
oh, thanks very much @frederick-vs-ja
I'm wondering why there are 2 build bots failed.  

Is `// UNSUPPORTED: c++03` a special flag to skip some bots?

And more, should I also add the `license header` to it?